addons/l10n_tr_nilvera_einvoice/models/account_edi_xml_ubl_tr.py

```diff
@@ -55,13 +72,23 @@ def _add_invoice_header_nodes(self, document_node, vals):
         document_node.update({
             'cbc:CustomizationID': {'_text': 'TR1.2'},
             'cbc:ProfileID': {
-                '_text': 'TEMELFATURA' if invoice.partner_id.l10n_tr_nilvera_customer_status == 'einvoice' else 'EARSIVFATURA'
+                '_text': (
```

Comment:
Please keep it as is, one liner

Reply:
It would be too much in a single line with the new code. We are already breaching the [PEP-8](https://peps.python.org/pep-0008/#maximum-line-length) limit. Let's make an intermediate variable if that's okay? By the way what's the issue with multi-line?

Reply:
Yes, regarding this point, you don't need the else; call the function like this.

```suggestion
                '_text': self._get_tr_profile_id(invoice)
```

since _get_tr_profile_id already handles all the cases, please check it.